### PR TITLE
CLOUDP-146560: Add support to arm64

### PR DESCRIPTION
**Pull request by helderjs**
### All Submissions:

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,9 +19,11 @@ COPY config/ config/
 COPY hack/ hack/
 
 ARG VERSION
+ARG TARGETOS
+ARG TARGETARCH
```
Comment (igor-karpukhin) [inline Dockerfile:23]: Shouldn't the Makefile be modified as well?

Comment (helderjs) [reply Dockerfile:23]: @igor-karpukhin If you see the ENVs below, it gets the value of these arguments and the envs are used in the Makefile to build for proper OS/ARCH. With that, no changes are needed in it.

File: .github/actions/certify-openshift-images/entrypoint.sh [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+  preflight check container "${DIGEST}" --docker-config="${HOME}/.docker/config.json"
+
+  # Send results to RedHat if preflight finished without errors
+  preflight check container "${DIGEST}" \
+    --submit \
+    --pyxis-api-token="${RHCC_TOKEN}" \
+    --certification-project-id="${RHCC_PROJECT}" \
+    --docker-config="${HOME}/.docker/config.json"
+done
+
```
Comment (igor-karpukhin) [inline .github/actions/certify-openshift-images/entrypoint.sh:21]: Too many empty lines. Everything else looks good!

**Review by igor-karpukhin — APPROVED** [review]

**Review by Sugar-pack — APPROVED** [review]
Great work!

**Review by fabritsius — APPROVED** [review]
Looks Great 👍

**Pull request merged**